packages/token/Scarb.toml

```diff
@@ -48,6 +49,8 @@ build-external-contracts = [
     "openzeppelin_test_common::mocks::account::DualCaseAccountMock",
     "openzeppelin_test_common::mocks::erc721::DualCaseERC721ReceiverMock",
     "openzeppelin_test_common::mocks::erc1155::DualCaseERC1155ReceiverMock",
+    "openzeppelin_test_common::mocks::erc2981::ERC2981OwnableMock",
+    "openzeppelin_test_common::mocks::erc2981::ERC2981AccessControlMock",
```

Comment:
We shouldn't need to build this if we use the COMPONENT_STATE pattern.

Reply:
Removed after refactoring to COMPONENT_STATE